build/theme.js

```diff
@@ -20,6 +20,16 @@ function buildThemeCss () {
     return `.border-${name} { border-color: ${theme.colors[name]}; }`
   })
 
+  const hover = Object.keys(theme.colors).map(name => {
+    return `.hover-${name} { transition: .2s ease color; }
```

Comment:
`color` should be `${theme.colors[name]}`

Reply:
What? Isn't it already?

![image](https://user-images.githubusercontent.com/5447088/36158683-cdc65bfa-10d4-11e8-8deb-14df539f06f9.png)

Reply:
😕 I totally misread it! you are right. 👍 

As much as possible, single purpose atoms should do 1 thing, so this kind of thing might be expressed as

```css
.transition-color {
  transition: .2s ease color;
}
.hover-red:hover { color: red;}
```

Reply:
or just start with a single

```css
.transition-all {
  transition: .2s ease all;
}
```
until you find a need more fine-grained control.